action/src/run.ts

```diff
@@ -163,4 +163,6 @@ export const run = async () => {
     ...context.repo
   });
   await createGithubComment();
+
+  setFailed('Visual regression differences found while taking a screenshot!');
```

Comment:
Super nit pick, could we make the failure log the same as the description of the commit status?
```suggestion
  setFailed('A visual regression was detected. Check Comparadise!');
```